recipe/meta.yaml

```diff
@@ -25,14 +25,18 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
+    - altgraph >=0.17
+    - macholib >=1.16
+    - modulegraph >=0.19
```

Comment:
Curious why these are needed in `host`? Typically this should only contain things that are used at build time (so `pip`, `python`, maybe `setuptools` or `cython`)

Reply:
Azure was failing and the error message was that `setup.py` could not find `macholib`, so I added them to `host`.

It turns out that In `v0.28` a `setup_requires` line was added to the `setup.py` file.
```
    install_requires=["altgraph>=0.16", "modulegraph>=0.17", "macholib>=1.16"],
    setup_requires=["altgraph>=0.16", "modulegraph>=0.17", "macholib>=1.16"],
```
https://github.com/ronaldoussoren/py2app/blob/162d923062a04ab623b546f77759aa8f87660910/setup.py#L294-L295

I was surprised too since this package generally updates smoothly.

The upstream repository removed `setup.py` and put everything into the `pyproject.toml` file, so the next version update will probably take a little more effort.

Reply:
Also, thanks for resolving the `macholib` update! I can also help maintain that feedstock since it is from the same developer and seems to be tightly integrated with this package.

Reply:
Ah ok. Odd that they include those in `setup_requires`, but adding to `host` makes sense in that case.

That would be great! Feel free to send a PR adding yourself there :)